internal/driver/driver_test.go

```diff
@@ -398,6 +399,18 @@ func (testFetcher) Fetch(s string, d, t time.Duration) (*profile.Profile, string
 		for _, s := range p.Sample {
 			s.NumLabel["bytes"] = append(s.NumLabel["bytes"], tags...)
 		}
+	case "heap_tags":
+		p = heapProfile()
+		sizeTags := []int64{2, 4, 8, 16, 32, 64, 128, 256}
```

Comment:
I think that the common usage of numeric tags like "bytes" in the heap traces is that each sample has just one value for the given tag, recording separate samples as needed. Otherwise it would not be possible to break the allocations down precisely. The test should probably be reflective of that.